src/dynarec/rv64/dynarec_rv64_helper.h

```diff
@@ -76,6 +76,29 @@
 // Write back ed in wback (if wback not 0)
 #define WBACK       if(wback) {SDxw(ed, wback, fixedaddress); SMWRITE();}
 
+// GETEB will use i for ed, and can use r3 for wback.
+#define GETEB(i, D) if(MODREG) {                \
+                    if(rex.rex) {               \
+                        wback = xRAX+(nextop&7)+(rex.b<<3);     \
+                        wb2 = 0;                \
+                    } else {                    \
+                        wback = (nextop&7);     \
+                        wb2 = (wback>>2)*8;     \
+                        wback = xRAX+(wback&3); \
+                    }                           \
+                    MV(i, wback);               \
+                    ANDI(i, i, 0xff);           \
+                    if (wb2) SLLI(i, i, wb2);   \
```

Comment:
I think the logic should be the oposite:
SRLI first, than AND.

Reply:
Yes, you're right.

Reply:
Fixed and force-pushed.